python/lsst/sims/photUtils/Bandpass.py

```diff
@@ -201,15 +201,15 @@ def readThroughput(self, filename, wavelen_min=None, wavelen_max=None, wavelen_s
         # Filename is single file, now try to open file and read data.
         try:
             if filename.endswith('.gz'):
-                f = gzip.open(filename, 'r')
+                f = gzip.open(filename, 'rt')
```

Comment:
Given that this has been a problem for a year and we only just noticed it, should we add a unit test to read in dummy files and verify that `wavelen`, `flambda` and `sb` (as the case may be) get set to the expected values?

Reply:
Oh, whoops - sorry, just saw that and i got too eager to press the big green button.
Yes you're right, we probably should.

Reply:
Um ... let's just make a ticket to write unit tests for Bandpass, as it seems like they don't exist at all.